salt/beacons/btmp.py

```diff
@@ -70,7 +70,7 @@ def beacon(config):
             __context__[LOC_KEY] = fp_.tell()
             pack = struct.unpack(FMT, raw)
             event = {}
-            for ind in range(len(FIELDS)):
+            for ind in enumerate(FIELDS):
```

Comment:
Oh, your right. it does. I forgot about that. What do you suggest here? leave it as range(len())?